.github/workflows/scripts/addClosingMessage.js

```diff
@@ -0,0 +1,75 @@
+// @ts-check
+
+/**
+ * @param {Object} params
+ * @param {import("@actions/core")} params.core
+ * @param {ReturnType<import("@actions/github").getOctokit>} params.github
+ * @param {import("@actions/github").context} params.context
+ */
+module.exports = async ({ core, context, github }) => {
+  try {
+    const owner = context.repo.owner;
+    const repo = context.repo.repo;
+    const issueNumber = context.issue.number;
+
+    const issue = await github.rest.issues.get({
+      owner,
+      repo,
+      issue_number: issueNumber,
+    });
+
+    const repositoryMap = {
```

Comment:
This mapping between GitHub repository slugs and the product id / product category id looks wrong. Source of truth https://github.com/mui/material-ui-docs/blob/92ff1fc67c1c999a9c92bdeb0f3ce6bbc173ee58/docs/src/modules/utils/getProductInfoFromUrl.ts. Suggestions below to match

Reply:
Most of the suggestions given make sense, but the linked code does not contain all of the possible combinations in terms of the `repo` -> `product` relation.

Should we define something (like an actual mapping) or keep it as is (with the adjustments from below) for now?

Reply:
> all of the possible combinations in terms of the repo -> product relation.

Do you have an example of a missing combination? I'm not completely following your point.

Reply:
Its the more fine-grained mapping of x-data-grid[-pro/-premium], x-charts, etc since we only have one repository and cannot determine which product is affected based on the limited information we have